src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -4520,6 +4520,10 @@ class SymbolTableVisitor : public CommonVisitor<SymbolTableVisitor> {
         // We skip this in the SymbolTable visitor, but visit it in the BodyVisitor
     }
 
+    void visit_While(const AST::While_t &/*x*/) {
+        // We skip this in the SymbolTable visitor, but visit it in the BodyVisitor
+    }
```

Comment:
I think this failed in main because previously there was no `visit_While()` in `SymbolTableVisitor`. It seems the `visit_While()` for `SymbolTableVisitor` was recently added in https://github.com/lcompilers/lpython/commit/47bd6d539bb062d8133e222e8d08b5610db4a466. It appears that there was no test case added for global while. I guess we could add a test case, even a minor one like in this PR should be good. I think this would help in ensuring that the global while does not break with future changes.